khal/khalendar/backend.py

```diff
@@ -451,24 +451,18 @@ def get_localized(self, start, end, minimal=False):
         start = aux.to_unix_time(start)
         end = aux.to_unix_time(end)
         if minimal:
-            sql_s = (
-                'SELECT events.calendar FROM '
-                'recs_loc JOIN events ON '
-                'recs_loc.href = events.href AND '
-                'recs_loc.calendar = events.calendar WHERE '
-                '(dtstart >= ? AND dtstart <= ? OR '
-                'dtend > ? AND dtend <= ? OR '
-                'dtstart <= ? AND dtend >= ?) AND events.calendar in ({0}) '
-                'ORDER BY dtstart')
+            sql_s = 'SELECT events.calendar FROM events'
```

Comment:
Ugh, it looks like this line is **never** executed during tests. Nothing broken when manually testing, but we need to increase coverage.